draft-wussler-openpgp-pqc.md

```diff
@@ -125,6 +125,19 @@ informative:
         ins: R. Davis
         name: Richard Davis
     date: August 2020
+  NIST-SP800-185:
+    target: https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-185.pdf
+    title: NIST Special Publication 800-185 – SHA-3 Derived Functions – cSHAKE, KMAC, TupleHash and ParallelHash
```

Comment:
```suggestion
    title: SHA-3 Derived Functions: cSHAKE, KMAC, TupleHash, and ParallelHash
```

Reply:
Aligned to the other NIST citations (without repeating "NIST Special...") we have, and aligned to the original title.

Reply:
"NIST Special Publication 800-185" is the essential part of the title, without it the reference is formally not complete (and be just guessed via our label, which is not sufficient and overly lengthy due to holding the whole string, I would say) omitting it is in my opinion out of the question. I make a different proposal though, following how the crypto-refresh handles this.

Reply:
Ok, I'm fine with this. Then we should also align the other reference to NIST SP 800 56C, so that it is homogeneous.